conf/env.in

```diff
@@ -5,6 +5,7 @@ POSTGRES_PASSWORD="${POSTGRES_PASSWORD}"
 POSTGRES_DB="${POSTGRES_DB}"
 SRID="${SRID}"
 LANGUAGES="${LANGUAGES}"
+LANGUAGE_CODE="${LANGUAGE_CODE}"
```

Comment:
I'm not sure we want to fix LANGUAGE_CODE in settings because Geotrek logic is to determine language with browser accept language or language defined i nsession with language menu